prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/OH/OH-receiver-transform.yml

```diff
@@ -0,0 +1,73 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+  OBX: '/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION/OBX'
+
+elements:
+  - name: oh-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: oh-sending-application-namespace-id
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ '%{MSH}-3-1' ]
+
+  - name: oh-sending-application-universal-id
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ '%{MSH}-3-2' ]
+
+  - name: oh-sending-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-3-3' ]
+
+  - name: oh-sending-facility-namespace-id
+    value: [ '"CDC PRIME"' ]
+    hl7Spec: [ '%{MSH}-4-1' ]
+
+  - name: oh-sending-application-universal-id
+    value: [ '"6DSMP9999"' ]
```

Comment:
line 33 should be 36DSMP9999.  Note, you forget number 3

Reply:
Thank you! Good catch!